tests/integration/charms/testdb/charmcraft.yaml

```diff
@@ -23,6 +23,7 @@ actions:
     params:
       mode:
         type: string
+    additionalProperties: true
```

Comment:
Apparently swapping the default value here is a deliberate breaking change, albeit one that is not yet documented. Being explicit seems a reasonable thing to do anyway.

See: https://github.com/juju/juju/issues/21294